src/lib/y2storage/filesystems/blk_filesystem.rb

```diff
@@ -132,6 +132,13 @@ def btrfs_subvolumes
         storage_subvols.to_a.map { |vol| BtrfsSubvolume.new(vol) }
       end
 
+      def in_network?
+        disks = ancestors.find_all { |d| d.is?(:disk) }
+        disks.append(self) if is?(:disk)
+        return true if disks.any? { |d| d.network_trasporter? }
+        false
```

Comment:
Instead of
```ruby
return true if disks.any? { |d| d.network_trasporter? }
false
```

You could simply have this, which is way more straightforward
```ruby
disks.any?(&:network_transporter?)
```